pandas/core/internals/blocks.py

```diff
@@ -1950,6 +1973,24 @@ def shift(self, periods, axis=0):
     def _ftype(self):
         return getattr(self.values, '_pandas_ftype', Block._ftype)
 
+    def _unstack(self, unstacker_func, new_columns, n_rows, fill_value):
+        dummy_arr = np.arange(n_rows)
```

Comment:
can you add a doc-string (or does it share)?

Reply:
The inherited one OK. I'm going to leave a comment explaining why we override.